install-ruby

```diff
@@ -9,9 +9,8 @@ case $(uname -m) in
   *) echo "Error: Your CPU $(uname -m) is not supported." >&2; exit 1 ;;
 esac
 
-if test -w /home/linuxbrew || sudo -v; then
+if test -w /home/linuxbrew || sudo mkdir -p /home/linuxbrew; then
```

Comment:
I like this change. Thanks! Could you please add `sudo -p` to add a custom prompt?
```suggestion
if test -w /home/linuxbrew || /usr/bin/sudo -p "[sudo] Enter password for $USER to install Ruby: " mkdir -p /home/linuxbrew; then
```

Reply:
This seems good, included.